cloud-aws-common/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/common/cost/AwsPricingCache.java

```diff
@@ -30,20 +34,41 @@ public class AwsPricingCache {
 
     private static final String PRICING_API_ENDPOINT_REGION = "us-east-1";
 
+    private static final String AWS_STORAGE_PRICING_JSON_LOCATION = "cost/aws-storage-pricing.json";
+
     private static final int CACHE_MAX_SIZE = 100;
 
     private static final int CACHE_RETENTION_IN_MINUTES = 5;
 
-    @Inject
-    private AWSPricing awsPricing;
+    private static final int HOURS_IN_30_DAYS = 720;
 
     private final Cache<PricingCacheKey, PriceListElement> cache;
 
+    private final Map<String, Map<String, Double>> storagePricingCache;
+
+    @Inject
+    private AWSPricing awsPricing;
+
     public AwsPricingCache() {
         this.cache = CacheBuilder.newBuilder()
                 .maximumSize(CACHE_MAX_SIZE)
                 .expireAfterAccess(Duration.ofMinutes(CACHE_RETENTION_IN_MINUTES))
                 .build();
+        this.storagePricingCache = loadStoragePriceList();
+    }
+
+    private static Map<String, Map<String, Double>> loadStoragePriceList() {
+        ClassPathResource classPathResource = new ClassPathResource(AWS_STORAGE_PRICING_JSON_LOCATION);
+        if (classPathResource.exists()) {
+            try {
+                String json = FileReaderUtils.readFileFromClasspath(AWS_STORAGE_PRICING_JSON_LOCATION);
+                return JsonUtil.readValue(json, new TypeReference<>() {
+                });
+            } catch (IOException e) {
+                throw new RuntimeException("Failed to load AWS storage price json!", e);
+            }
+        }
+        throw new RuntimeException("Failed to load AWS storage price json!");
```

Comment:
You should use a different error message.

Reply:
I'll fix it in my PR from branch https://github.com/hortonworks/cloudbreak/tree/costfixes